server/cmd/code-annotation/handler/assignments.go

```diff
@@ -0,0 +1,74 @@
+package handler
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/render"
+
+	"github.com/src-d/code-annotation/server/cmd/code-annotation/serializer"
+	"github.com/src-d/code-annotation/server/model"
+	"github.com/src-d/code-annotation/server/repository"
+)
+
+func HandleGetAssignmentsForUserExperiment(w http.ResponseWriter, r *http.Request) {
+	experimentID, err := strconv.Atoi(chi.URLParam(r, "experimentId"))
+	if err != nil {
+		render.Render(w, r, serializer.NewErrors(err))
+		return
+	}
+
+	userID, err := strconv.Atoi("1")
+	if err != nil {
+		render.Render(w, r, serializer.NewErrors(err))
+		return
+	}
+
+	assignments, err := repository.GetAssignmentsFor(userID, experimentID)
```

Comment:
As I remember here we create all assignments. How do you think is it better to put creation logic here or in a service? And keep repository as a slim interface to DB?

Reply:
I was thinking in:
```go
assignments, err := repository.GetAssignmentsFor(userID, experimentID)
if err.is(NO_ASSIGNMENTS_INITIALIZED) {
    assignments, err = repository.CreateAssignmentsFor(userID, experimentID)
} 
```
I agree on putting business logic in `service`, instead of in `repository`... but should we create a new `service` package for only these 3 lines that could be in the `handler.GetAssignmentsForUserExperiment` ?

Reply:
I would put it in handler as you posted snippet above for now. Later if we have more cases we can refactor it to service.